### PR TITLE
Add simple table component

### PR DESCRIPTION
**Pull request by bangunbagustapa**
#### Changes

Add simple table component

#### Preview

Screen Shot 2021-07-15 at 10 39 16

### CODE REVIEW

File: src/components/JdsSimpleTable/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import JdsSimpleTable from './SimpleTable.vue'
+
+export { JdsSimpleTable }
+export default JdsSimpleTable
```
Comment (adzharamrullah) [inline src/components/JdsSimpleTable/index.js:4]: **Low Priority**
It's better to be given a new line

File: src/components/JdsSimpleTable/SimpleTable.stories.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          <td>Data Table</td>
+          <td>Data Table</td>
+        </tr>
+      </tbody>
+    </jds-simple-table>
+    `,
+  }
+}
+
+export const Default = Template.bind({})
```
Comment (adzharamrullah) [inline src/components/JdsSimpleTable/SimpleTable.stories.js:45]: Low Priority
It's better to be given a new line

File: src/components/JdsSimpleTable/SimpleTable.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    td {
+      padding: 6px 8px;
+      border-top: 1px solid V.$gray-200;
+    }
+
+    &:nth-child(even) {
+      background-color: V.$gray-50;
+    }
+  }
+}
```
Comment (adzharamrullah) [inline src/components/JdsSimpleTable/SimpleTable.scss:34]: Low Priority
It's better to be given a new line

File: src/components/JdsSimpleTable/index.js (hunk 1 above)
Comment (Ibwedagama) [reply src/components/JdsSimpleTable/index.js:4]: thankyou, will fix it asap

File: src/components/JdsSimpleTable/SimpleTable.stories.js (hunk 2 above)
Comment (Ibwedagama) [reply src/components/JdsSimpleTable/SimpleTable.stories.js:45]: thankyou, will fix it asap

File: src/components/JdsSimpleTable/SimpleTable.scss (hunk 3 above)
Comment (Ibwedagama) [reply src/components/JdsSimpleTable/SimpleTable.scss:34]: thankyou, will fix it asap

**Review by Arif9878 — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

File: src/components/JdsSimpleTable/index.js (hunk 1 above)
Comment (bangunbagustapa) [reply src/components/JdsSimpleTable/index.js]: Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)

File: src/components/JdsSimpleTable/SimpleTable.stories.js (hunk 2 above)
Comment (bangunbagustapa) [reply src/components/JdsSimpleTable/SimpleTable.stories.js]: Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)

File: src/components/JdsSimpleTable/SimpleTable.scss (hunk 3 above)
Comment (bangunbagustapa) [reply src/components/JdsSimpleTable/SimpleTable.scss]: Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)

**Review by adzharamrullah — APPROVED** [review]

**Pull request merged**